tamproxy/comm/tamp_serial.py

```diff
@@ -44,7 +44,8 @@ def get_port(self):
         else:
             print "Enter the index of the desired serial port:"
             for i,p in enumerate(ports): print i,p
-            return ports[int(raw_input())]
+            return ports[0]
+            # return ports[int(raw_input())]
```

Comment:
This should probably go upstream, but not in this changeset - we've hit this problem elsewhere